### PR TITLE
feat(client):add july 2021 footer

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/i18n/locales/english/trending.json [hunk 1]
```diff
@@ -1,62 +1,63 @@
...
+  "article17link":"https://www.freecodecamp.org/news/if-function-excel-tutorial-and-how-to-do-multiple-if-statements-in-excel/",
+  "article18title":"HTML List",
+  "article18link":"https://www.freecodecamp.org/news/html-list-how-to-use-bullet-points-ordered-and-unordered-lists/",
+  "article19title":"Wav File",
+  "article19link":"https://www.freecodecamp.org/news/wav-file-format-how-to-open-a-wav-and-convert-wavs-to-mp3/",
+  "article20title":"Model View Controller ",
+  "article20link":"https://www.freecodecamp.org/news/the-model-view-controller-pattern-mvc-architecture-and-frameworks-explained/",
+  "article21title":"Git Checkout Remote Branch",
+  "article21link":"https://www.freecodecamp.org/news/git-checkout-remote-branch-tutorial/",
+  "article22title":"Insert checkbox in word",
```
Comment (ShaunSHamilton) [inline client/i18n/locales/english/trending.json:46]:
```suggestion
  "article22title":"Insert Checkbox in Word",
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Out of curiosity, why does this say "July" footer?

File: client/i18n/locales/english/trending.json (hunk 1 above)
Comment (ahmadabdolsaheb) [reply client/i18n/locales/english/trending.json:46]: it is a convention we have with the editorial team.

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**